charts/kubescape-operator/values.yaml

```diff
@@ -93,6 +94,21 @@ global:
     enabled: false
     createEgressRules: false
 
+# Continuous scanning configurations
+continuousScanning:
+  configMapName: cs-matching-rules
+
+  # Matching rules for the monitored resources.
+  # Kubescape will watch resources of every provided GVR across the provided
+  # namespaces.
+  matchingRules:
```

Comment:
@vladklokun  I'm not sure why this should be in the values file.
We don't want users to change it...

Reply:
I added this to the `values.yaml` file so that _we_ can change the monitored more conveniently. I wouldn’t overestimate how often people change the deeply-buried configurables in Helm charts.